.github/workflows/fpb-lint.yml

```diff
@@ -14,7 +19,88 @@ jobs:
       with:
         node-version: '16.x'
     - run: npm install -g free-programming-books-lint
-    - run: fpb-lint ./books/
-    - run: fpb-lint ./casts/
-    - run: fpb-lint ./courses/
-    - run: fpb-lint ./more/
+      
+    - name: Push
+      if: ${{ github.event_name == 'push' }}
+      run: |
+        fpb-lint ./books/
+        fpb-lint ./casts/
+        fpb-lint ./courses/
+        fpb-lint ./more/
+
+    - name: Pull Request
+      if: ${{ always() && 
+          github.event_name == 'pull_request' }}
+      run: |
+        fpb-lint ./books/ &>> output.log || echo "Analyzing..."
+        fpb-lint ./casts/ &>> output.log || echo "Analyzing..."
+        fpb-lint ./courses/ &>> output.log || echo "Analyzing..."
+        fpb-lint ./more/ &>> output.log || echo "Analyzing..."
+
+        cat > error.log
```

Comment:
Is this line to create a empty file that can be used at line 60?, Better use `touch` I tought

```shell
touch error.log
```

Reply:
You're right, create a code suggestion